src/ripple/consensus/Consensus.cpp

```diff
@@ -84,7 +84,11 @@ shouldCloseLedger(
 }
 
 bool
-checkConsensusReached(std::size_t agreeing, std::size_t total, bool count_self)
+checkConsensusReached(
+    std::size_t agreeing,
+    std::size_t total,
+    bool count_self,
+    std::size_t minConsensusPct)
```

Comment:
just so I understand - what's the motivation for passing `minConsensusPct` directly and not the `parms` object like you do in other places?